go/toolchains.rst

```diff
@@ -36,17 +36,22 @@ The SDK
 At the bottom is the Go SDK. This is the same thing you would get if you go to the main
 `Go website`_ and download a `binary distribution`_.
 
-The go_sdk_ rule is responsible for downloading these, and adding just enough of a build file
-to expose the contents to Bazel. It currently also builds the cross compiled standard libraries
-for specific combinations, although we hope to make that an on demand step in the future.
+This is always bound to ``@go_sdk`` and can be referred to directly if needed, but in general
+you should always access it through the toolchain.
+
+The go_download_sdk_, go_host_sdk_ and go_local_sdk_ family of rules are responsible for downloading
+these, and adding just enough of a build file to expose the contents to Bazel.
+It currently also builds the cross compiled standard libraries for specific combinations, although
+we hope to make that an on demand step in the future.
 
 SDKs are specific to the host they are running on and the version of Go they want to use
 but not the target they compile for. The Go SDK is naturally `cross compiling`_.
 
-The Go rules already adds a go_sdk_ rule for all the host and Go version pairs that are shipped
-on the main Go language website, so there should be no need to declare one of these unless you
-need a `forked version of Go`_\, however you may want to `control the version`_ or use the
-`installed sdk`_.
+If you don't do anything special, the go rules will already add a go_sdk for you, defaulting to the
```

Comment:
If you don't do anything special, the Go rules will download the most recent official SDK for your host.